docs: identification headers (#9241)
Unleash/unleash: website/docs/reference/sdks/index.mdx

-   [React Proxy SDK](/docs/generated/sdks/client-side/react.md)
-   [Svelte Proxy SDK](/docs/generated/sdks/client-side/svelte.md)
-   [Vue Proxy SDK](/docs/generated/sdks/client-side/vue.md)

> ### SDK Identification Headers

Reviewer: ```suggestion
### SDK identification headers
```

> To identify which frontend and backend SDK instances are connected to Unleash, we have standardized three headers that should be attached to all requests made by the SDK to the server (`/api/client` for backend SDKs and `/api/frontend` for frontend SDKs):

Reviewer: ```suggestion
To identify which frontend and backend SDK instances are connected to Unleash, the following three headers must be attached to all requests made by the SDK to the server (`/api/client` for backend SDKs and `/api/frontend` for frontend SDKs):
```

> - **`unleash-appname`**: Specifies the name of your application that is communicating with Unleash. This name is user-defined with some SDKs providing a default fallback value. For example, `billing-service`.

Reviewer: ```suggestion
- `unleash-appname`: Specifies the name of your application that is communicating with Unleash. This name is user-defined with some SDKs providing a default fallback value. For example, `billing-service`.
```

> - **`unleash-connection-id`**: A unique identifier for the current SDK instance, automatically generated using the idiomatic unique identifier generator for the specific programming ecosystem (e.g., `randomUUID` from `crypto` in Node.js). This connection ID represents a long-lived connection between the SDK and the Unleash server. An SDK should generate a new connection ID each time an SDK instance is instantiated (e.g., when calling `new Unleash()` in a Node SDK). The same connection ID should be used for each request when an SDK instance polls the API. Additionally, there is a legacy `unleash-instanceid` header that is not implemented by all SDKs and can be overwritten by the client, which is the main reason `unleash-connection-id` was introduced.

Reviewer: ```suggestion
- `unleash-connection-id`: A unique identifier for the current SDK instance, automatically generated using the idiomatic unique identifier generator for the specific programming ecosystem (for example, `randomUUID` from `crypto` in Node.js). This connection ID represents a long-lived connection between the SDK and the Unleash server. An SDK should generate a new connection ID each time an SDK instance is instantiated (for example, when calling `new Unleash()` in a Node SDK). The same connection ID should be used for each request when an SDK instance polls the API. Additionally, there is a legacy `unleash-instanceid` header that is not implemented by all SDKs and can be overwritten by the client, which is the main reason `unleash-connection-id` was introduced.
```

> - **`x-unleash-sdk`**: Provides information about the language/framework/version making the request. The format is the language/platform followed by a colon and the semantic version of the SDK (e.g., `unleash-client-java@10.0.1` or `unleash-client-node@6.4.4`). Each SDK implementation should use a platform-idiomatic method to read the version from a package manager and update it with each version release.

Reviewer: ```suggestion
- `x-unleash-sdk`: Provides information about the language/framework and version of the SDK making the request. The format is the language/framework followed by a colon and the semantic version of the SDK (for example, `unleash-client-java@10.0.1` or `unleash-client-node@6.4.4`). Each SDK implementation should use a platform-idiomatic method to read the version from a package manager and update it with each version release.
```
